compute_tools/src/compute.rs

```diff
@@ -467,7 +439,9 @@ impl ComputeNode {
 
         let pg = self.start_postgres(spec.storage_auth_token.clone())?;
 
-        self.apply_config(&compute_state)?;
+        if spec.spec.mode == ComputeMode::Primary {
+            self.apply_config(&compute_state)?;
+        }
```

Comment:
I think we should also wrap `handle_*` part in the `reconfigure` in the same way. We currently don't have read replicas in the cloud and we don't reload their config, but we may want in the future